P5/Source/Specs/lb.xml

```diff
@@ -32,7 +32,7 @@
   </classes>
   <content><empty/></content>
   <exemplum xml:lang="en">
-    <p>This example shows typographical line breaks within metrical lines, where they occur at different
+    <p>This example shows topographical line beginning within metrical lines, where they occur at different
```

Comment:
At the very least, I think “beginning” should be plural.
But I am not really fond of using it as a noun like that at all. How about “This example shows the encoding of the beginning of each topographic line as it occurs in both the 1664 and 1667 editions” or some such?

Reply:
Agreed! It definitely should have been plural, that is entirely a slip on my part. I also agree with your rewording of the sentence.